### PR TITLE
Game View Class

### PR DESCRIPTION
**Pull request by german-e-mas**
In this PR I made a new class that encompasses all view elements.

Before, the AerodromeView was listening to the Game's callbacks. Now, a new GameView class does so.

This class contains the previous AerodromeView and some elements, like the score field, a game over legend and a restart button. They are passed to the GameView in the MainActivity.

The GameView listens to changes in the AerodromeView, like screen touches. The Aerodrome also tells the GameView when it's scale values are set and it is ready to start.

Like before, the Controller listens to changes made into the GameView, and tells the Game to modify an aircraft or start the game.

Unused callbacks were removed.

ptal @pablisho

Screenshot:
device-2017-03-07-121235

### CODE REVIEW

File: app/src/main/java/mas/german/landingplanes/Game.java [hunk 1]
```diff
@@ -109,14 +114,6 @@
...
          */
         void onAircraftOutsideAerodrome(int id);
-
-        /**
-         * A crash happened.
-         *
-         * @param firstId   ID of the first Aircraft involved in the crash.
-         * @param secondId  ID of the second Aircraft involved in the crash.
-         */
-        void onCrash(int firstId, int secondId);
```
Comment (pablisho) [inline app/src/main/java/mas/german/landingplanes/Game.java]: Why did you remove this?

File: app/src/main/java/mas/german/landingplanes/MainActivity.java [hunk 2]
```diff
@@ -2,20 +2,30 @@
...
         setContentView(R.layout.activity_main);
-        mView = (AerodromeView) findViewById(R.id.view_aerodrome);
-        mView.setController(mController);
+
+        // Pass the layout elements to the View Class.
+        mGameView = new GameView(getApplicationContext());
+        mGameView.setAerodrome((AerodromeView) findViewById(R.id.view_aerodrome));
+        mGameView.setScoreText((TextView) findViewById(R.id.score_field));
+        mGameView.setGameOverText((TextView) findViewById(R.id.game_over_text));
+        mGameView.setRestartButton((Button) findViewById(R.id.btn_restart));
```
Comment (pablisho) [inline app/src/main/java/mas/german/landingplanes/MainActivity.java]: I think all of these should be part of the contructor instead of setting them. The Gameview cannot live without them.

File: app/src/main/java/mas/german/landingplanes/Game.java (hunk 1 above)
Comment (german-e-mas) [inline app/src/main/java/mas/german/landingplanes/Game.java]: It was not being used after all. It's implementation was empty. The game over sequence occurs in the _onGameOver_ method.

File: app/src/main/java/mas/german/landingplanes/MainActivity.java (hunk 2 above)
Comment (german-e-mas) [inline app/src/main/java/mas/german/landingplanes/MainActivity.java]: You are absolutely right. I'll modify this.

**Comment by german-e-mas** [comment]
Thanks for the review @pablisho ! I addressed the comments about the `GameView` contructor. I'll proceed to merge this.
